cmd/demo-client/main.go

```diff
@@ -35,9 +35,9 @@ var (
 	buildDate = "unknown"
 )
 
-var urlFlag = &cli.StringFlag{
-	Name:  "url",
-	Usage: "root URL for fetching randomness",
+var urlsFlag = &cli.StringSliceFlag{
+	Name:  "urls",
```

Comment:
Personally I’d leave this as “url” but suggest in the usage text that you can use the parameter multiple times. “urls” suggests to me that it could be comma separated, when actually you can only use a single url for each usage of the parameter.

Reply:
good call. i'll fix that.